pkg/question/question.go

```diff
@@ -382,39 +394,68 @@ func AskInstanceType(h *ec2helper.EC2Helper, defaultInstanceType string) (*strin
 		}
 	}
 
-	question := "Instance Type (eg. m5.xlarge, c5.xlarge)"
+	question := "Which instance type should by used? (eg. m5.xlarge, c5.xlarge)"
+	instanceValidation := func(h *ec2helper.EC2Helper, instanceType string) bool {
+		for _, instance := range instanceTypes {
+			if *instance.InstanceType == instanceType {
+				return true
+			}
+		}
+		return false
+	}
 
-	answer := AskQuestion(&AskQuestionInput{
+	model := &questionModel.PlainText{}
+	err = questionModel.AskQuestion(model, &questionModel.QuestionInput{
 		QuestionString: question,
-		DefaultOption:  defaultOption,
-		StringOptions:  stringOptions,
+		DefaultOption:  *defaultOption,
+		EC2Helper:      h,
+		Fns:            []questionModel.CheckInput{instanceValidation},
 	})
 
+	if err != nil {
+		return nil, err
+	}
+
+	answer := model.GetTextAnswer()
 	return &answer, nil
 }
 
 // Ask the users to enter instance type vCPUs
-func AskInstanceTypeVCpu() string {
-	question := "vCPUs (integer)"
+func AskInstanceTypeVCpu(h *ec2helper.EC2Helper) (string, error) {
+	question := "How many vCPUs are to be used?"
```

Comment:
Ideas for rewriting the vCPU and memory questions with active voice:

* How many vCPUs do you want?
* How many vCPUs do you need?

Reply:
I'll address this in the next PR when I reformat the questions